src/node/waku_node.py

```diff
@@ -186,3 +255,47 @@ def is_nwaku(self):
 
     def is_gowaku(self):
         return "go-waku" in self.image
+
+    def parse_rln_credentials(self, default_args, is_registration):
+        rln_args = {}
+        keystore_path = None
+
+        creds_f = open(default_args["rln-creds-source"])
+
+        imported_creds = json.load(creds_f)
+        selected_id = default_args["rln-creds-id"]
+
+        if len(imported_creds) < 4 or any(value is None for value in imported_creds.values()):
```

Comment:
can we add a log line saying what happened here if we return in this case

Reply:
Add at https://github.com/waku-org/waku-interop-tests/pull/16/commits/6f8a5c7da43424f1ed7b53d72c0bafdc92573922